cypress/cypress/e2e/submariner_install.cy.js

```diff
@@ -0,0 +1,44 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
```

Comment:
This test should have the following name to be executed in the correct order:
`03_submariner_install.cy.js`